test/verifyImageLabels.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
```

Comment:
I think this file can be just moved into the generic `test/0.0` folder and run for all builds.

Reply:
I put it here because currently all tests under test/[version] are running inside docker ("kafka" or "cp-kafkacat" image). 
but this test's verifying the labels of kafka image itself so I think that the labels should be checked outside.
Also by calling from script in .travis.yml, this should already run for all builds.

Reply:
As per the comment below, it could just be added as another container (using the mounted docker socket) and perform these checks in a consistent way so that anybody can run the tests. However, for now I think this is fine. I'll fix up at a later date if it becomes an issue. However, agreed that it is different from the functionality tests that are currently containerized.